digital-credentials/identity-create.tentative.https.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<title>Digital Identity Credential tests.</title>
+<link rel="help" href="https://wicg.github.io/digital-credentials/" />
+<script src="/common/get-host-info.sub.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<body>
+  <iframe id="same-origin"></iframe>
+  <iframe id="cross-origin"></iframe>
+</body>
+
+<script type="module">
+  import { sendMessage, loadIframe } from "./support/helper.js";
+
+  const iframeSameOrigin = document.querySelector("iframe#same-origin");
+  const iframeCrossOrigin = document.querySelector("iframe#cross-origin");
+  const mediations = ["silent", "optional", "conditional", "required"];
+
+  promise_setup(async () => {
+    const hostInfo = get_host_info();
+    await Promise.all([
+      loadIframe(
+        iframeCrossOrigin,
+        `${hostInfo.HTTPS_REMOTE_ORIGIN}/digital-credentials/support/iframe.html`
+      ),
+      loadIframe(iframeSameOrigin, "/digital-credentials/support/iframe.html"),
+    ]);
+  });
+
+  promise_test(async (t) => {
+    const result = await navigator.identity.create();
+    assert_equals(result, null);
+  }, "navigator.identity.create() frame just returns null.");
+
```

Comment:
I don't think that its worthwhile to test navigator.identity.create() in iframe given that navigator.identity.create() in a toplevel frame should fail.

Reply:
As currently spec'ed, the DC spec calls into the [default create operation](https://www.w3.org/TR/credential-management-1/#dom-credential-create-slot) which resolves with `null`. My reading is that null is returned in both cases.

Reply:
I agree that the spec calls for the return value to be null in this scenario.
I am unsure whether the <iframe> test has value given that create() fails for the toplevel origin, but the <iframe> test doesn't hurt, so I am ok with it.